### DISCUSSION TITLE
Tasks in a RESTARTING state not moving to RETRY

### DISCUSSION THREAD

**Discussion by kyle-inv-disco** [apache/airflow#25157] [Q&A]
Hi everyone,

I have a situation whereby currently running tasks when a TriggerDagRunOperator is called on a (running) Dag with reset DagRun equal to true (resetting all the tasks on the running instance) then shift to a RESTARTING (purple) state, and stay there indefinitely.

Is this expected behaviour, and how then are tasks marked as RESTARTING expected to retry/restart without manually clearing them?

Kind regards,
Kyle

**Comment by potiuk** [comment]
No. It should not happen. You need to describe it in detail including version,, gather all the evidences (logs, configratiom, deployment details) and open an issue.
